### DISCUSSION TITLE
Ignore accents and diacritics

### DISCUSSION THREAD

**Discussion by tuvah** [darlal/obsidian-switcher-plus#78] [Ideas]
Search for filename or heading containing a character with diacritics e.g. "îïĩí" is matched only with exactly same character. Could the search match with simple "i"?

This feature is already requested for core Obsidian switcher: https://forum.obsidian.md/t/ignore-accents-diacritics-in-search/1655/

**Comment by darlal** [comment]
Hey @tuvah Switcher++ currently uses the core fuzzy search algorithm, so the behavior of Switcher++ and the core builtin switcher is the same in this case. I know this comes up often in the discord and forum (as you've linked), so hopefully it will get addressed soon in the core search algorithm.
